events/cmd/root.go

```diff
@@ -18,6 +20,14 @@ func EventCmd(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "events",
 		Short: "Prints out akash events in real time",
+		PreRunE: func(cmd *cobra.Command, _ []string) error {
+			cmd.Flags().String(flags.FlagNode, "", "The node address")
```

Comment:
i think you want to do this outside of `PreRunE`?

Reply:
right, that was rather stupid